File: content/29/a.go
package main
import "time"
func main() {
   n := int64(366 * 24 * 60 * 60)
   o := time.Unix(n, 0)
   s := o.Format("Mon Jan 2 2006")
   println(s == "Fri Jan 1 1971")
}

File: content/29/a.php
<?php
$n = 365 * 24 * 60 * 60;
# example 1
$s = strftime('%a %b %#d %Y', $n);
# example 2
$s2 = strftime('%a %b %#e %Y', $n);
# print
var_dump($s == 'Fri Jan 1 1971', $s2 == 'Fri Jan 1 1971');
